apps/web/src/utils/paymasterConfig.ts

```diff
@@ -0,0 +1,24 @@
+import { baseSepolia } from "viem/chains";
+import { ENTRYPOINT_ADDRESS_V06 } from "permissionless";
+import { createClient, createPublicClient, http } from "viem";
+import { paymasterActionsEip7677 } from "permissionless/experimental";
+
+
+
+export function initializeClient() {
+  
+  const client = createPublicClient({
+    chain: baseSepolia,
+    transport: http(),
+  });
+  
+  return client;
+}
+
+
+const paymasterService = 'https://api.developer.coinbase.com/rpc/v1/base-sepolia/1IhTcPOmhK5aEq-4WqRZMJoOh0oPenD2'; // Change this to the official paymaster
```

Comment:
is this comment still relevant?

Reply:
Yes, will update this when the paymaster team sets up the official paymaster for this project. Using this url only for testing.

Reply:
can you make sure that we have a task in Linear to address anything like this (and reference the task # in the comment as well)